src/formio/components/FileField.js

```diff
@@ -184,12 +185,8 @@ class FileField extends Formio.Components.components.file {
 
   upload(files) {
     if (this.component.multiple && this.component.maxNumberOfFiles) {
-      // this.data.file contains files that may have already been uploaded, while the argument 'files' gives the
```

Comment:
`data` is a mapping of component keys -> values, so `file` was only relevant if it was the actual key of the file component